etc/testing/gatekeeper.yaml

```diff
@@ -2117,39 +2118,6 @@ metadata:
   name: gatekeeper-admin
   namespace: gatekeeper-system
 ---
-apiVersion: policy/v1beta1
```

Comment:
I think we might need to keep this (allows us to test against security configurations of customers), but I think the API just changed to policy/v1 maybe?

Reply:
I think we are keeping the new equivalent way of doing it, but PSP was deprecated. https://kubernetes.io/docs/concepts/security/pod-security-policy/ 

The `pod-security.kubernetes.io/enforce: restricted` namespace annotation along with tweaks like `seccompProfile` are the replacement. I tried to follow the [migration plan](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/) and set it to the most restrictive policy.